### DISCUSSION TITLE
Clearing errors in any field

### DISCUSSION THREAD

**Discussion by xArt2150** [TanStack/form#1031] [Q&A]
Hello, we have an edge case where we need to clear the errors of a set of fields in the change callback of another field.

Something like this:

```
const onChange = (value: TData) => {
  field.form.setErrorMap({
    'field1': undefined,
    'field2': undefined
  });
  field.handleChange(value);
}
```

Is there a way to do this correctly? The example above doesn't work, and if I understand the feature released here (https://github.com/TanStack/form/issues/786), using setErrorMap allows us to only set/clear form-level error and the current field-level error.

**Comment by SimonEvertree** [comment]
Have you tried setFieldMeta?
